pointer_go119.go

```diff
@@ -59,3 +62,8 @@ func (p *Pointer[T]) Swap(val *T) (old *T) {
 func (p *Pointer[T]) CompareAndSwap(old, new *T) (swapped bool) {
 	return p.p.CompareAndSwap(old, new)
 }
+
+// String returns a human readable representation of a Pointer's underlying value.
+func (p *Pointer[T]) String() string {
+	return fmt.Sprint(p.p.Load())
+}
```

Comment:
So, the way we're using build tags here is a bit non-standard.
We have a separate 1.18 and 1.19 file, where 1.18 has nothing shared with 1.19.

With build tags, "go1.18" is actually enabled for all Go versions starting 1.18.
(Which is why we have a "!go1.19" in the 1.18 file.)

I think what would be better to do here is sharing this String method between 1.18 and 1.19 somehow.

We need it to be defined in a file with the "go1.18" tag, but not the "!go1.19" tag.
So perhaps we can rename pointer_go118 to pointer_go118_pre119, making it clear that it's only for Go 1.18,
and create a new pointer_go118 file that only has the "go1.18" tag and this method.
That fill will compile for both, 1.18 and 1.19.

Reply:
Done! 
This pointer to a  pointer test case that's failing is a bit tricky though. Any suggestions? The test file is so clean as is, I don't want to overcomplicate it. I not sure if it's worth recursively loading in the case of pointers either.